### ISSUE TITLE
Add and style text about "Who We Are"

### ISSUE THREAD

**Issue by JavaScriptErika** [CascadingCoders/LandingPage#1]
### Please include the following text and style:

We're a welcoming community of developers of all skill levels who have a love or interest in creative code. Even if you've never heard of it before or it sounds intimidating - this is the place for you!

From CSS web animation to experimenting with HTML5 Canvas, code is art and a means of self-expression that can be translated into effective UI/UX and bring about unforgettable experiences and interaction to the web. We curate content/resources, hold CSS/SVG art (creative code) challenges, and inspire each other to become better Cascading Coders!

👥 Join our Community
[Facebook Group](https://www.facebook.com/groups/1752972351391061/)

❤️ Like/Follow:
[Twitter](https://twitter.com/cascadingcoders)
[Facebook Page ](https://www.facebook.com/CascadingCoders/)

**Comment by JavaScriptErika** [comment]
You may claim this issue, but please wait, as we are pending design. I will tag you when we're ready to proceed.

**Comment by thisavijit** [comment]
please assign this task to me!

**Comment by JavaScriptErika** [comment]
You've got it @adavijit - will let you know when we're ready to proceed!

**Comment by thisavijit** [comment]
@JavaScriptErika  done! please check

**Comment by JavaScriptErika** [comment]
@adavijit Thank you for the update in the README. I should have clarified the issue better as it's for the landing page. Is this a task you'd like to create in the project as a component? I have the wireframe that you can work on for that!

**Comment by brunagomes** [comment]
@JavaScriptErika I can work on this!

**Comment by thisavijit** [comment]
@JavaScriptErika Yes, please provide the wireframe and give some time to me to complete

**Comment by JavaScriptErika** [comment]
Hi @brunagomes thank you for your enthusiasm! This issue has already been claimed. I see you have interest in another issue that is unclaimed, so I will assign you there.

Sounds good @adavijit Please take a look at issue #6 and I also will provide the image below for the component you need to create. I am looking for the "About Us" content to be placed in as an <About /> component. Placeholder images and text is ok for the rest - no css styling needed just looking for the structure (bulma structure with columns / containers etc).

There is nothing on the page yet (as of now until we get some merged PRs), so it's ok if your component is the only one showing!

![Capture](https://user-images.githubusercontent.com/19316487/66776486-7068d600-ee94-11e9-85d2-6fb10ad6927e.PNG)

**Comment by thisavijit** [comment]
@JavaScriptErika sent PR! #9

**Comment by JavaScriptErika** [comment]
@brunagomes would you like to work on this?
